WalletWasabi.Fluent/Views/Wallets/Settings/WalletCoinJoinSettingsView.axaml

```diff
@@ -40,15 +40,15 @@
               Height="37"
               Command="{Binding SelectEconomicalSettings}"
               Classes.selected="{Binding EconomicalProfileSelected}"
-              ToolTip.Tip="Only participate in coinjoins during the cheapest parts of the week.">
+              ToolTip.Tip="Minimizes transaction fees and allows non-private coin consolidation.">
         <StackPanel Spacing="10" Orientation="Horizontal">
           <PathIcon Opacity="0.9" Foreground="{DynamicResource FunctionButtonForegroundColor}" Data="{StaticResource coinjoin_cost}" />
           <TextBlock Margin="0 2 0 0" Text="Reduce Costs" />
         </StackPanel>
       </Button>
     </StackPanel>
 
-    <DockPanel ToolTip.Tip="Out of the registered coins into coinjoin only one can be non-private (anonymity score 1).">
+    <DockPanel ToolTip.Tip="Only submits one non-private coin in each round.">
```

Comment:
One can easily misunderstand this that only one coin is registered in each round. This is confusing.